usd/src/usd_parser/usd_model/WorldInterface.hh

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef USD_PARSER_USD_MODEL_WORLD_INTERFACE_HH_
+#define USD_PARSER_USD_MODEL_WORLD_INTERFACE_HH_
+
+#include <ostream>
+#include <string>
+#include <vector>
+
+#include <ignition/math/Vector3.hh>
+
+#include "sdf/sdf_config.h"
+
+namespace sdf
+{
+  // Inline bracket to help doxygen filtering.
+  inline namespace SDF_VERSION_NAMESPACE {
+  //
+  namespace usd
+  {
+    /// \brief This class stores data about the world
+    class WorldInterface {
+      public:
```

Comment:
nit: Can you update the formatting for this class. `public: ` on each member. If all the members are going to be public, then maybe a `struct` is better?.